docs/installation.rst

```diff
@@ -157,35 +157,57 @@ properly.
+|                               |                                 |  instance.                                                                             |
 +-------------------------------+---------------------------------+----------------------------------------------------------------------------------------+
 |                               |                                 | Hashed password to access protected endpoints. If left empty, all administrative       |
 | ADMIN_PASSWORD                |  ``""``                         | tasks are disabled.                                                                    |
-|                               |                                 | To generate the proper password HASH, use ``ihatemoney generate_password_hash``        |
-|                               |                                 | and copy the output into the value of *ADMIN_PASSWORD*.                                |
+|                               |                                 |                                                                                        |
+|                               |                                 | **Production value:** To generate the proper password HASH, use                        |
+|                               |                                 | ``ihatemoney generate_password_hash`` and copy the output into the value of            |
+|                               |                                 | *ADMIN_PASSWORD*.                                                                      |
 +-------------------------------+---------------------------------+----------------------------------------------------------------------------------------+
-| ALLOW_PUBLIC_PROJECT_CREATION |  ``True``                       | If set to `True`, everyone can create a project without entering the admin password    |
-|                               |                                 | If set to `False`, the password needs to be entered (and as such, defined in the       |
+| ALLOW_PUBLIC_PROJECT_CREATION |  ``True``                       | If set to ``True``, everyone can create a project without entering the admin password  |
+|                               |                                 | If set to ``False``, the password needs to be entered (and as such, defined in the     |
 |                               |                                 | settings).                                                                             |
+|                               |                                 |                                                                                        |
+|                               |                                 | **Production value:** Usually, you will want to set it to ``False`` for a personal     |
```

Comment:
We have different profile for the admins: some of them will have a personal instance, some of them will offer a service for others. I don't think this should be a recommended setting for production value. (we should provide no hint for this)